### ISSUE TITLE
Pytest raises 600+ warnings

### ISSUE THREAD

**Issue by cclauss** [kornia/kornia#1225]
## 🐛 Bug

$ pytest
```
-- Docs: https://docs.pytest.org/en/stable/warnings.html
= 19994 passed, 280 skipped, 29 xfailed, 27 xpassed, 603 warnings in 635.21s (0:10:35) =
```

Often these are about the depreciation of capabilities that may soon be modified or deleted.  Researching and proposing fixes for the can be a **useful activity for new contributors**.

**Comment by edgarriba** [comment]
@shijianjian can we start removing some of the known warnings ?

**Comment by shijianjian** [comment]
Working on it now. #1281

We gonna remove all the deprecation warnings and deprecated functions. For the JIT warnings, we will keep it as it is.

**Comment by shijianjian** [comment]
Closed by #1281. Only 100 warnings left, mostly raised by JIT.

**Comment by cclauss** [comment]
Nice! Thx.
